src/PlanList.tsx

```diff
@@ -106,12 +114,12 @@ export class PlanList extends React.Component<Plans, IState>{
                           primary={planObject.name}
                           secondary={planObject.item_uid.substr(0,8)}/>
                         <ListItemSecondaryAction>
-                          <IconButton onClick={() => this.handleMoveForward(planObject.item_uid)} edge="end" aria-label="comments">
-                            <KeyboardArrowUpIcon />
-                          </IconButton>
-                          <IconButton onClick={() => this.handleMoveBackward(planObject.item_uid)} edge="end" aria-label="comments">
-                            <KeyboardArrowDownIcon />
-                          </IconButton>
+                          {(index != 0) && <IconButton onClick={() => this.handleDecrement(index)} edge="end" aria-label="comments">
```

Comment:
Shouldn't this be `!==` and in the case below?
```suggestion
                          {(index !== 0) && <IconButton onClick={() => this.handleDecrement(index)} edge="end" aria-label="comments">
```